core/src/components/datetime/datetime.tsx

```diff
@@ -2045,6 +2051,7 @@ export class Datetime implements ComponentInterface {
 
     return (
       <ion-picker-column
+        aria-label="Select a day period"
```

Comment:
Open to other ideas for naming here. Wasn't sure what else to call the AM/PM picker

Reply:
I'm ok with the naming here.